docs/tutorial.md

```diff
@@ -176,3 +180,53 @@ spec:
           serviceName: frontend
           servicePort: 80
 ```
+
+## Adding Health Probes to your service
+By default, Ingress controller will provision a HTTP GET probe for the exposed pods.  
+The probe properties can customized by adding a Readiness or a Liveness Probe to your deployment/pod spec.
```

Comment:
The probe properties can be customized by adding a [Readiness] or a [Liveness] probe to your deployment/pod spec.

Could you add reference to the Readiness and Liveness probe documentation.